### PR TITLE
Optional parameter to show or hide date in tooltip title

### PR DESCRIPTION
**Pull request by martinmanzo**
## Description
Added the property _shouldShowDateInTitle_ to the tooltip module. It default to true to enable backwards compatibility.

## Motivation and Context
In order to use the grouped bar chart in non date scenarios we needed to make optional the visibility of the date. We included a property to show it or hide it accordingly.

## How Has This Been Tested?
Added tests for the accesor and for the updateTitle function.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: docs/scripts/demo-line.js [hunk 1]
```diff
@@ -13334,10 +13335,14 @@ webpackJsonp([5,10],[
...
-	                tooltipTitleText = title + ' - ' + formatDate(date);
+	            if (shouldShowDateInTitle) {
+	                var date = new Date(dataPoint[dateLabel]),
+	                    tooltipTitleText = title + ' - ' + formatDate(date);
 	
-	            tooltipTitle.text(tooltipTitleText);
+	                tooltipTitle.text(tooltipTitleText);
+	            } else {
+	                tooltipTitle.text(title);
+	            }
```
Comment (Golodhros) [inline docs/scripts/demo-line.js]: Nit: I would do something slightly different:
  let tooltipTitle = title;
  
  if (shouldShowDateInTitle) {
    tooltipTitle = `${tooltipTitle} - ${formatDate(new Date(dataPoing[dateLabel]))}`;
  }
  tooltipTitle.text(tooltipTitle);

Comment (martinmanzo) [inline docs/scripts/demo-line.js]: Sure!
First time I see that ${..} notation, how does it work?

Comment (Golodhros) [inline docs/scripts/demo-line.js]: They are the ES2015 template literals, you can check more here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

thanks a lot!

Comment (martinmanzo) [inline docs/scripts/demo-line.js]: Done! You are welcome!

**Comment by Golodhros** [comment]
Thanks a lot @martinmanzo !!

**Pull request merged**